modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -1,49 +1,43 @@
-[id="procedure-module-guidelines"]
-= Procedure Module Guidelines
+[id="con-procedure-module-guidelines_{context}"]
 
-The required parts of a procedure module are a procedure and its introduction. Optionally, the module can also include prerequisites and additional resources.
+= Procedure module guidelines
+
+Procedure modules explain how to do something. They must include a title, a brief introduction, and one or more imperative statements. Procedure modules can also contain prerequisites, verification steps, and additional resources or next steps.
 
 .Schema of a procedure module
 image::procedure-diagram.png[]
 
-[discrete]
-== Procedure Introduction
-The introduction is a short description of the procedure. For example, it can be a lead-in sentence or an infinitive phrase (_To extract the certificate: <steps>_). See also _The IBM Style Guide_ footnoteref:[ibm-style-guide,DERESPINIS, Francis, Peter HAYWARD, Jana JENKINS, Amy LAIRD, Leslie McDONALD, Eric RADZINKSI. _The IBM style guide: conventions for writers and editors_. Upper Saddle River, NJ: IBM Press/Pearson, c2012. ISBN 0132101300.] for details on introducing procedures.
+.Procedure Title
+Start the title of a procedure module with a verb or a gerund, such as Create or Creating.
```

Comment:
Same question as before about verb vs. gerund. I thought we were clearly requiring the gerund.